src/greedy.jl

```diff
@@ -31,34 +31,57 @@ Greedy reduced basis assembling strategy.
   See also [`estimate_error`](@ref)
 - `tol::Float64=1e-3`: tolerance for error estimate, below which the assembly is terminated.
 - `n_truth_max::Int=64`: maximal number of truth solves to be taken up in the basis.
-- `init_from_rb::Bool=true`: if `true`, uses initial guesses from RB eigenvectors.
+- `Ψ_init::Function=rb_guess`: returns initial guess for truth solver.
   See also [`interpolate`](@ref).
 - `verbose::Bool=true`: print information during assembly if `true`.
 - `exit_checks::Bool=true`: if `false`, no exit checks will be performed and the assembly
   will run until `tol` or `n_truth_max` are reached.
 """
 @kwdef struct Greedy
     estimator::ErrorEstimate
-    tol::Float64       = 1e-3
-    n_truth_max::Int   = 64
-    init_from_rb::Bool = true  # TODO: More general mechanism
-    verbose::Bool      = true
-    exit_checks::Bool  = true
+    tol::Float64      = 1e-3
+    n_truth_max::Int  = 64
+    Ψ_init::Function  = rb_guess
```

Comment:
Putting this as `::Function` here is pointless as `::Function` is not a concrete type. I'd leave the type off.

Reply:
Yes agreed, that would make more sense.